nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/SpaceToDepth.java

```diff
@@ -65,6 +66,16 @@ public SpaceToDepth(INDArray in, INDArray out, int blockSize, String dataFormat)
         addIArgument(blockSize, isNHWC ? 1 : 0);
     }
 
+
+
+    public SpaceToDepth(INDArray x, int blockSize, DataFormat dataFormat) {
+        super(null, x,null,null,null);
+        this.blockSize = blockSize;
+        this.dataFormat = dataFormat.toString();
+
+    }
```

Comment:
As per DepthToSpace, missing addIArgument, but better to use `this(x, null, blockSize, dataFormat)`